transmissions/models.py

```diff
@@ -85,6 +85,8 @@ class Status(EnumDict):
 
     status = models.IntegerField(default=Status.CREATED)
 
+    exception = models.TextField(blank=True, default='')
```

Comment:
Thanks @pbaranay! Unfortunately, the `Notification` model is meant to be extremely lean since it's storing 100s of millions of entries in production. 

I'd suggest moving this feature to separate table since most notifications shouldn't have any exception anyway!